examples/demo/pages/login.vue

```diff
@@ -83,31 +83,34 @@ export default {
     async login() {
       this.error = null
 
-      return this.$auth
-        .loginWith('local', {
-          data: {
-            username: this.username,
-            password: this.password
-          }
-        })
-        .catch(e => {
-          this.error = e.response.data + ''
-        })
+      try {
+        await this.$auth
+          .loginWith('local', {
+            data: {
+              username: this.username,
+              password: this.password
+            }
+          })
+
+      } catch(e) {
+        this.error = e + ''
```

Comment:
Just see that this line has't been merged. Can you change to `this.error = e.response.data + ''` please :)